WebRoot/public/zimbrax.jsp

```diff
@@ -0,0 +1,14 @@
+<%@ page buffer="8kb" autoFlush="true" %>
+<%@ page pageEncoding="UTF-8" contentType="text/html; charset=UTF-8" %>
+<%@ page import="java.util.*,javax.naming.*,com.zimbra.client.ZAuthResult" %>
+<%
+	
+    ZAuthResult authResult = (ZAuthResult) request.getAttribute("authResult");
+    pageContext.setAttribute("csrfToken", authResult.getCsrfToken());
+%>
+<script TYPE="text/javascript">
+    localStorage.setItem("csrfToken" , "${csrfToken}");
+    var url = window.location.origin + "/zimbrax/index.html"
```

Comment:
can we add a TODO comment here, that we need to remove index.html part when jetty configs are in place